lib/clowne/adapters/sequel/resolvers/after_persist.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'clowne/adapters/sequel/specifications/after_persist_does_not_support'
+
+module Clowne
+  module Adapters # :nodoc: all
+    class Sequel
+      module Resolvers
+        class AfterPersist
+          def self.call(_source, record, _declaration, **)
+            operation = Clowne::Adapters::Sequel::Operation.current
+            operation.add_after_persist(
+              proc do
+                raise Specifications::AfterPersistDoesNotSupportException
```

Comment:
@palkan or we can raise the exception immediately when user will try to use `after_persist` (don't store it in operation) and not give a chance to use cloner. wdyt?
```
def self.call(_source, record, _declaration, **)
  raise Specifications::AfterPersistDoesNotSupportException
end
```

Reply:
> we can raise the exception immediately

yeah, makes sense